FirebaseSessions/Sources/SessionInitiator.swift

```diff
@@ -22,8 +26,43 @@ import Foundation
 ///      and comes to the foreground.
 ///
 class SessionInitiator {
+  let sessionTimeout: Double = 1800
+  let now: () -> Date
```

Comment:
I might rename `now` to something more function-ey. Maybe `getNow`?

Reply:
I would make it more clearer and unambiguous - Can we call it `getCurrentTime`?

Reply:
How do y'all feel about @ncooke3 's comment of just something like `currentTimeProvider` or `dateProvider`? This would be consistent with iCore's style like this iCore class [HeartbeatController](https://github.com/firebase/firebase-ios-sdk/blob/f2e44defd3886a735199f1ec6768377adbaa5a4d/FirebaseCore/Internal/Sources/HeartbeatLogging/HeartbeatController.swift#L53:L64)